examples/mint-burn-oft-adapter/contracts/mocks/SuperChainMintBurnERC20.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { IMintableBurnable } from "@layerzerolabs/oft-evm/contracts/interfaces/IMintableBurnable.sol";
+
+interface IERC7802 {
+    event CrosschainBurn(address indexed from, uint256 amount, address indexed sender);
+    event CrosschainMint(address indexed to, uint256 amount, address indexed sender);
+
+    function crosschainBurn(address _from, uint256 _amount) external;
+    function crosschainMint(address _to, uint256 _amount) external;
+}
+
+contract SuperChainMintBurnERC20 is ERC20, IERC7802, IMintableBurnable, Ownable {
```

Comment:
Why is this in `examples` instead of `oft-evm`?  Shouldn't it be in `oft-evm` so folks don't need to copy + paste to use it?